### DISCUSSION TITLE
Error when trying to create a new organization

### DISCUSSION THREAD

**Discussion by shounakacharya** [Sunbird-Ed/Community#106] [Installation] [answered]
System Configuration Details:
1.	KP, DP, DB, Yarn, Jenkins, Other-Services: Ubuntu 18.04 LTS, 4vCPU, 16GB RAM, 250 GB HDD
2.	Google Kubernetes engine: 1.21.12-gke.2200 with 4 worker nodes. Note: Currently Sunbird is not compatible with K8s APIs  >= 1.22 and we have checked this
3.	All systems are running on GCP
4. Learner Service Github Tag: release-4.6.0_RC4

@gandham-santhosh and  @keshavprasadms

When I am trying to create the root org on the deployed sunbird instance, I am getting the following error. This output is from postman:

The API that was called was:

http://sunbird.enter-mnemon.com/api/org/v1/create

{
    "id": "api.org.create",
    "ver": "v1",
    "ts": "2022-08-25 14:32:04:970+0000",
    "params": {
        "resmsgid": null,
        "msgid": "0053d41d1f51bc76445b65135d0df59d",
        "err": "MANDATORY_PARAMETER_MISSING",
        "status": "MANDATORY_PARAMETER_MISSING",
        "errmsg": "Mandatory parameter organisationType is missing."
    },
    "responseCode": "CLIENT_ERROR",
    "result": {}
}

On looking at the learning service logs I am getting the following error:

{"timestamp":"2022-08-25T14:29:55.608Z","msg":"Original Url: /service/health","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-4","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:29:55.608Z","msg":"Original Captcha: null","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-4","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:29:55.608Z","msg":"Original Url: /service/health","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-2","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:29:55.609Z","msg":"Original Captcha: null","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-2","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:00.150Z","msg":"Original Url: /v1/org/create","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-4","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:00.150Z","msg":"Original Captcha: null","lname":"modules.OnRequestHandler","tname":"application-akka.actor.default-dispatcher-4","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:00.383Z","msg":"{eid='LOG', edata={level=trace, requestid=c51d1365d7b6408c66f6289a22da6528, type=system, message=ENTRY LOG: method : POST, url: /v1/org/create , For Operation : createOrg, params=[{orgName=Sunbird, isTenant=true, channel=sunbird, description=Default Channel, id=null, userId=null, organisationType=board}]}}","lname":"util.PrintEntryExitLog","tname":"application-akka.actor.default-dispatcher-4","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:00.469Z","msg":"Cassandra query : SELECT * FROM sunbird.user WHERE id=?;","lname":"org.sunbird.cassandraimpl.CassandraOperationImpl","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:00.470Z","msg":"OrgManagementActor: Create org method call start","lname":"org.sunbird.learner.actors.OrganisationManagementActor","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:00.764Z","msg":"ElasticSearchRestHighImpl:search: calling search for index org_alias, with query = {\"from\":0,\"size\":250,\"query\":{\"bool\":{\"must\":[{\"term\":{\"isTenant.raw\":{\"value\":true,\"boost\":1.0}}},{\"term\":{\"channel.raw\":{\"value\":\"sunbird\",\"boost\":1.0}}}],\"adjust_pure_negative\":true,\"boost\":1.0}},\"_source\":{\"includes\":[],\"excludes\":[]}}","lname":"org.sunbird.common.ElasticSearchRestHighImpl","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.046Z","msg":"ConnectionManager:createRestClient client initialisation done. ","lname":"org.sunbird.helper.ConnectionManager","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.074Z","msg":"ELASTIC SEARCH CONNECTION ESTABLISHED for restClient from EVN with Following Details cluster null  hostName10.128.0.52 port 9300,9300,9300true","lname":"org.sunbird.helper.ConnectionManager","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.281Z","msg":"ElasticSearchRestHighImpl:search: calling search for index org_alias, with query = {\"from\":0,\"size\":250,\"query\":{\"bool\":{\"must\":[{\"term\":{\"isTenant.raw\":{\"value\":true,\"boost\":1.0}}},{\"term\":{\"slug.raw\":{\"value\":\"sunbird\",\"boost\":1.0}}}],\"adjust_pure_negative\":true,\"boost\":1.0}},\"_source\":{\"includes\":[],\"excludes\":[]}}","lname":"org.sunbird.common.ElasticSearchRestHighImpl","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.351Z","msg":"start call for registering the channel for org id ==0136105052802908160","lname":"org.sunbird.learner.util.Util","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.352Z","msg":"Util:registerChannel: Channel registration request data = {\"request\":{\"channel\":{\"code\":\"0136105052802908160\",\"name\":\"sunbird\",\"description\":\"Default Channel\"}}}","lname":"org.sunbird.learner.util.Util","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.506Z","msg":"end call for channel registration for org id ==0136105052802908160","lname":"org.sunbird.learner.util.Util","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.579Z","msg":"Exception occurred while inserting record to organisation : Undefined column name organisationtype","lname":"org.sunbird.cassandraimpl.CassandraOperationImpl","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"ERROR","stack_trace":"<#d8f77af7> com.datastax.driver.core.exceptions.InvalidQueryException: Undefined column name organisationtype\n\tat com.datastax.driver.core.Responses$Error.asException(Responses.java:181)\n\tat com.datastax.driver.core.SessionManager$4.apply(SessionManager.java:250)\n\tat com.datastax.driver.core.SessionManager$4.apply(SessionManager.java:219)\n\tat com.google.common.util.concurrent.Futures$ChainingListenableFuture.run(Futures.java:906)\n\tat com.google.common.util.concurrent.Futures$1$1.run(Futures.java:635)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat com.datastax.shaded.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\n\tat java.lang.Thread.run(Thread.java:829)\nWrapped by: <#71d987f8> com.datastax.driver.core.exceptions.InvalidQueryException: Undefined column name organisationtype\n\tat com.datastax.driver.core.exceptions.InvalidQueryException.copy(InvalidQueryException.java:49)\n\tat com.datastax.driver.core.DriverThrowables.propagateCause(DriverThrowables.java:35)\n\tat com.datastax.driver.core.AbstractSession.prepare(AbstractSession.java:86)\n\tat org.sunbird.cassandraimpl.CassandraOperationImpl.insertRecord(CassandraOperationImpl.java:62)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.createOrg(OrganisationManagementActor.java:186)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.onReceive(OrganisationManagementActor.java:67)\n\tat org.sunbird.actor.core.BaseActor.onReceive(BaseActor.java:34)\n\tat akka.actor.UntypedAbstractActor$$anonfun$receive$1.applyOrElse(AbstractActor.scala:291)\n\tat akka.actor.Actor$class.aroundReceive(Actor.scala:517)\n\tat akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:180)\n\tat akka.actor.ActorCell.receiveMessage(ActorCell.scala:588)\n\tat akka.actor.ActorCell.invoke(ActorCell.scala:557)\n\tat akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258)\n\tat akka.dispatch.Mailbox.run(Mailbox.scala:225)\n\tat akka.dispatch....\n","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.580Z","msg":"Cassandra query : INSERT INTO sunbird.organisation(id,channel,createdBy,createdDate,description,orgName,organisationType,rootOrgId,slug,status,isTenant,isSSOEnabled,orgLocation) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?);","lname":"org.sunbird.cassandraimpl.CassandraOperationImpl","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.639Z","msg":"OrganisationManagementActor:createOrg: Error occurred = Invalid property organisationtype.","lname":"org.sunbird.learner.actors.OrganisationManagementActor","tname":"SunbirdMWSystem-rr-usr-dispatcher-85","level":"ERROR","stack_trace":"<#7e7a71d6> org.sunbird.common.exception.ProjectCommonException: Invalid property organisationtype.\n\tat org.sunbird.cassandraimpl.CassandraOperationImpl.insertRecord(CassandraOperationImpl.java:82)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.createOrg(OrganisationManagementActor.java:186)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.onReceive(OrganisationManagementActor.java:67)\n\tat org.sunbird.actor.core.BaseActor.onReceive(BaseActor.java:34)\n\tat akka.actor.UntypedAbstractActor$$anonfun$receive$1.applyOrElse(AbstractActor.scala:291)\n\tat akka.actor.Actor$class.aroundReceive(Actor.scala:517)\n\tat akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:180)\n\tat akka.actor.ActorCell.receiveMessage(ActorCell.scala:588)\n\tat akka.actor.ActorCell.invoke(ActorCell.scala:557)\n\tat akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258)\n\tat akka.dispatch.Mailbox.run(Mailbox.scala:225)\n\tat akka.dispatch.Mailbox.exec(Mailbox.scala:235)\n\tat akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)\n\tat akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)\n\tat akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)\n\tat akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)\n","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}
{"timestamp":"2022-08-25T14:30:01.759Z","msg":"ProducerConfig values: \n\tmetric.reporters = []\n\tmetadata.max.age.ms = 300000\n\treconnect.backoff.ms = 50\n\tsasl.kerberos.ticket.renew.window.factor = 0.8\n\tbootstrap.servers = [10.128.0.50:9092]\n\tssl.keystore.type = JKS\n\tsasl.mechanism = GSSAPI\n\tmax.block.ms = 0\n\tinterceptor.classes = null\n\tssl.truststore.password = null\n\tclient.id = learner-6bd6cbb8b8-mktbl-default-logback-relaxed\n\tssl.endpoint.identification.algorithm = null\n\trequest.timeout.ms = 30000\n\tacks = 0\n\treceive.buffer.bytes = 32768\n\tssl.truststore.type = JKS\n\tretries = 0\n\tssl.truststore.location = null\n\tssl.keystore.password = null\n\tsend.buffer.bytes = 131072\n\tcompression.type = none\n\tmetadata.fetch.timeout.ms = 60000\n\tretry.backoff.ms = 100\n\tsasl.kerberos.kinit.cmd = /usr/bin/kinit\n\tbuffer.memory = 33554432\n\ttimeout.ms = 30000\n\tkey.serializer = class org.apache.kafka.common.serialization.ByteArraySerializer\n\tsasl.kerberos.service.name = null\n\tsasl.kerberos.ticket.renew.jitter = 0.05\n\tssl.trustmanager.algorithm = PKIX\n\tblock.on.buffer.full = false\n\tssl.key.password = null\n\tsasl.kerberos.min.time.before.relogin = 60000\n\tconnections.max.idle.ms = 540000\n\tmax.in.flight.requests.per.connection = 5\n\tmetrics.num.samples = 2\n\tssl.protocol = TLS\n\tssl.provider = null\n\tssl.enabled.protocols = [TLSv1.2, TLSv1.1, TLSv1]\n\tbatch.size = 16384\n\tssl.keystore.location = null\n\tssl.cipher.suites = null\n\tsecurity.protocol = PLAINTEXT\n\tmax.request.size = 1048576\n\tvalue.serializer = class org.apache.kafka.common.serialization.ByteArraySerializer\n\tssl.keymanager.algorithm = SunX509\n\tmetrics.sample.window.ms = 30000\n\tpartitioner.class = class org.apache.kafka.clients.producer.internals.DefaultPartitioner\n\tlinger.ms = 15000\n","lname":"org.apache.kafka.clients.producer.ProducerConfig","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.840Z","msg":"ProducerConfig values: \n\tmetric.reporters = []\n\tmetadata.max.age.ms = 300000\n\treconnect.backoff.ms = 50\n\tsasl.kerberos.ticket.renew.window.factor = 0.8\n\tbootstrap.servers = [10.128.0.50:9092]\n\tssl.keystore.type = JKS\n\tsasl.mechanism = GSSAPI\n\tmax.block.ms = 0\n\tinterceptor.classes = null\n\tssl.truststore.password = null\n\tclient.id = learner-6bd6cbb8b8-mktbl-default-logback-relaxed\n\tssl.endpoint.identification.algorithm = null\n\trequest.timeout.ms = 30000\n\tacks = 0\n\treceive.buffer.bytes = 32768\n\tssl.truststore.type = JKS\n\tretries = 0\n\tssl.truststore.location = null\n\tssl.keystore.password = null\n\tsend.buffer.bytes = 131072\n\tcompression.type = none\n\tmetadata.fetch.timeout.ms = 60000\n\tretry.backoff.ms = 100\n\tsasl.kerberos.kinit.cmd = /usr/bin/kinit\n\tbuffer.memory = 33554432\n\ttimeout.ms = 30000\n\tkey.serializer = class org.apache.kafka.common.serialization.ByteArraySerializer\n\tsasl.kerberos.service.name = null\n\tsasl.kerberos.ticket.renew.jitter = 0.05\n\tssl.trustmanager.algorithm = PKIX\n\tblock.on.buffer.full = false\n\tssl.key.password = null\n\tsasl.kerberos.min.time.before.relogin = 60000\n\tconnections.max.idle.ms = 540000\n\tmax.in.flight.requests.per.connection = 5\n\tmetrics.num.samples = 2\n\tssl.protocol = TLS\n\tssl.provider = null\n\tssl.enabled.protocols = [TLSv1.2, TLSv1.1, TLSv1]\n\tbatch.size = 16384\n\tssl.keystore.location = null\n\tssl.cipher.suites = null\n\tsecurity.protocol = PLAINTEXT\n\tmax.request.size = 1048576\n\tvalue.serializer = class org.apache.kafka.common.serialization.ByteArraySerializer\n\tssl.keymanager.algorithm = SunX509\n\tmetrics.sample.window.ms = 30000\n\tpartitioner.class = class org.apache.kafka.clients.producer.internals.DefaultPartitioner\n\tlinger.ms = 15000\n","lname":"org.apache.kafka.clients.producer.ProducerConfig","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.842Z","msg":"Kafka version : 0.10.0.1","lname":"org.apache.kafka.common.utils.AppInfoParser","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.842Z","msg":"Kafka commitId : a7a17cdec9eaa6c5","lname":"org.apache.kafka.common.utils.AppInfoParser","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.738Z","msg":"{\"eid\":\"ERROR\",\"ets\":1661437801656,\"ver\":\"3.0\",\"mid\":\"c51d1365d7b6408c66f6289a22da6528\",\"actor\":{\"id\":\"e9991ab1-2369-4081-aae8-247d44ef15c0\",\"type\":\"User\"},\"context\":{\"channel\":\"ORG_001\",\"pdata\":{\"id\":\"dev.sunbird.learning.service\",\"pid\":\"learner-service\",\"ver\":\"3.9.0\"},\"env\":\"Organisation\",\"cdata\":[{\"id\":\"c51d1365d7b6408c66f6289a22da6528\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"err\":\"400\",\"stacktrace\":\"org.sunbird.cassandraimpl.CassandraOperationImpl.insertRecord(CassandraOperationImpl.java:82)\\norg.su\",\"errtype\":\"INVALID_PROPERTY_ERROR\"}}","lname":"TelemetryEventLogger","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.850Z","msg":"Invalid property organisationtype.","lname":"controllers.BaseController","tname":"Thread-11","level":"ERROR","stack_trace":"<#7e7a71d6> org.sunbird.common.exception.ProjectCommonException: Invalid property organisationtype.\n\tat org.sunbird.cassandraimpl.CassandraOperationImpl.insertRecord(CassandraOperationImpl.java:82)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.createOrg(OrganisationManagementActor.java:186)\n\tat org.sunbird.learner.actors.OrganisationManagementActor.onReceive(OrganisationManagementActor.java:67)\n\tat org.sunbird.actor.core.BaseActor.onReceive(BaseActor.java:34)\n\tat akka.actor.UntypedAbstractActor$$anonfun$receive$1.applyOrElse(AbstractActor.scala:291)\n\tat akka.actor.Actor$class.aroundReceive(Actor.scala:517)\n\tat akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:180)\n\tat akka.actor.ActorCell.receiveMessage(ActorCell.scala:588)\n\tat akka.actor.ActorCell.invoke(ActorCell.scala:557)\n\tat akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258)\n\tat akka.dispatch.Mailbox.run(Mailbox.scala:225)\n\tat akka.dispatch.Mailbox.exec(Mailbox.scala:235)\n\tat akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)\n\tat akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)\n\tat akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)\n\tat akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)\n","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.854Z","msg":"{\"eid\":\"ERROR\",\"ets\":1661437801851,\"ver\":\"3.0\",\"mid\":\"c51d1365d7b6408c66f6289a22da6528\",\"actor\":{\"id\":\"e9991ab1-2369-4081-aae8-247d44ef15c0\",\"type\":\"User\"},\"context\":{\"channel\":\"ORG_001\",\"pdata\":{\"id\":\"dev.sunbird.learning.service\",\"pid\":\"learner-service\",\"ver\":\"3.9.0\"},\"env\":\"Organisation\",\"cdata\":[{\"id\":\"c51d1365d7b6408c66f6289a22da6528\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"err\":\"INVALID_PROPERTY_ERROR\",\"stacktrace\":\"org.sunbird.cassandraimpl.CassandraOperationImpl.insertRecord(CassandraOperationImpl.java:82)org.sun\",\"errtype\":\"api_access\"}}","lname":"TelemetryEventLogger","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
{"timestamp":"2022-08-25T14:30:01.868Z","msg":"{eid='LOG', edata={level=trace, requestid=c51d1365d7b6408c66f6289a22da6528, type=system, message=EXIT LOG: method : POST, url: /v1/org/create , For Operation : createOrg, params=[{msgid=c51d1365d7b6408c66f6289a22da6528, errmsg=Invalid property organisationtype., resmsgid=null, err=INVALID_PROPERTY_ERROR, status=FAILED, responseCode=400}]}}","lname":"util.PrintEntryExitLog","tname":"Thread-11","level":"INFO","HOSTNAME":"learner-6bd6cbb8b8-mktbl","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT","uid":"e9991ab1-2369-4081-aae8-247d44ef15c0","op":"createOrg","appVer":null,"appId":null,"did":null,"sid":null,"reqId":"c51d1365d7b6408c66f6289a22da6528"}

**Comment by beepdot** [comment, marked as answer]
@amit-tarento @reshmi-nair Can you guys please share your thoughts on this error?

**Reply by shounakacharya** [reply]
@keshavprasadms, @amit-tarento @reshmi-nair there is also failed migration of Cassandra kesypace before mentioned in this thread https://github.com/Sunbird-Ed/Community/discussions/89. Initially it failed on version 1.86. I resolved according to the reply and now it is failing at 1.90. Maybe a reply there, would solve the problem here.

**Reply by amitec006** [reply]
@shounakacharya Its related to cassandra migration job (https://github.com/orgs/Sunbird-Ed/discussions/89) issue, once that fixed. this issue will get resolved automatically.

**Reply by shounakacharya** [reply]
@amit-tarento : Thanks. Updating the cassandra issue also solved this issue. Marking this as answered.

Thanks
Shounak
